src/Subaccount/Filter/Subaccount.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Vonage\Subaccount\Filter;
+
+use Vonage\Client\Exception\Request;
+use Vonage\Entity\Filter\FilterInterface;
+
+class Subaccount implements FilterInterface
```

Comment:
This is namespaced properly, but should make this Hungarian since we do that in most places.
```suggestion
class SubaccountFilter implements FilterInterface
```